### ISSUE TITLE
How can I do the contrary?

### ISSUE THREAD

**Issue by beppe9000** [tjanczuk/edge#295]
I'd like to embed node.js modules in my existing .net applications. Is there any way?

**Comment by w0lph** [comment]
Come on beppe9000, you can do better than that. Just search around the web and you will find your answer.

**Comment by beppe9000** [comment]
I am posting here as nobody seems to be interested in this (i've spent a fair share of time googling around) : seems just like nobody wants to host node.js in .net (or google won't show it to me).

So, you have done .Net -> Javascript. I ask you how is diffucult to go from Javascript to .Net ?

Right now I am thinking of implementing node-specific instructions things like Require and Exports with Jint or Jurassic and then load the modules, but I do not want to re-invent the weel.

**Comment by beppe9000** [comment]
I have spent a fair share of time searching prior to posting here, as I found nothing remotely close to that on the internet (maybe Google is not showing me o.O ). 

You have done .Net -> Node.js. Have you got any idea about the difficulty of going node.js -> .Net ?

Right now I am thinking of using Jint or Jurassic and adding support for node.js specificities like Require/Exports, and other things that could be needed, but I do not want to re-invent the wheel, so I came here asking.
